homeassistant/bootstrap.py

```diff
@@ -88,6 +88,16 @@
     # as possible so problem integrations can
     # be removed
     "frontend",
+    # Ensure the logbook integration platform helper is loaded
+    # before any logbook platforms since some legacy
+    # integration events (at least doorbird) use a different
+    # event type per device.
+    #
+    # Example: doorbird_device123_triggered
+    #
+    # Once we get rid of all
+    # the events per device, we can remove this.
+    "logbook",
```

Comment:
I don't like this, but I think its OK. since don't have a better way.  

I'm still investigating an alternate solution so I may back it back out shortly

Reply:
So we don't need this.  Its a strait up bug in doorbird

Reply:
Fixed in https://github.com/home-assistant/core/pull/74649